
string epc
string NewEPC
---
int64 epcUpdate
bool status

 
